inductiva/api/methods.py

```diff
@@ -428,36 +444,20 @@ def run_simulation(
     resource_pool_id: Optional[UUID] = None,
 ) -> pathlib.Path:
     """Run a simulation synchronously via Inductiva Web API."""
+    task = run_async_simulation(api_method_name, input_dir, params,
+                                resource_pool_id)
+    with task:
+        task.wait()
 
-    params = {
-        "sim_dir": input_dir,
-        **params,
-    }
-    type_annotations = {
-        "sim_dir": pathlib.Path,
-    }
-
-    result = invoke_api(
-        api_method_name,
-        params,
-        type_annotations,
-        output_dir=output_dir,
-        return_type=pathlib.Path,
-        resource_pool_id=resource_pool_id,
-    )
-
-    if not isinstance(result, pathlib.Path):
-        raise RuntimeError(f"Expected result to be a Path, got {type(result)}")
-
-    return result
+    return task.download_output(output_dir)
```

Comment:
This goes in hand with a discussion we are having on #362. But we can move forward and when we merge we update this.